osm_fieldwork/form_components/choice_fields.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2020, 2021, 2022, 2023, 2024 Humanitarian OpenStreetMap Team
+#
+# This file is part of OSM-Fieldwork.
+#
+#     This is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with OSM-Fieldwork.  If not, see <https:#www.gnu.org/licenses/>.
+#
+
+"""This creates DataFrames for choice lists used in the survey form.
+
+These include predefined options for fields such as yes/no responses
+and issues related to digitization problems. Each choice contains
+multilingual labels to support various languages.
+
+Returns:
+    tuple: Two pandas DataFrames containing the `choices` data for yes/no
+and digitisation problems respectively.
+"""
+
+import pandas as pd
+
+# Define the choices sheet
+choices_data = [
+    {"list_name": "yes_no", "name": "yes", "label::english(en)": "Yes"},
+    {"list_name": "yes_no", "name": "no", "label::english(en)": "No"},
+]
+
+choices_df = pd.DataFrame(choices_data)
+
+digitisation_choices = [
+    {
+        "list_name": "digitisation_problem",
+        "name": "lumped",
+        "label::english(en)": "Lumped - one polygon (more than one building digitized as one)",
+        "label::swahili(sw)": "Lumped - poligoni moja (zaidi ya jengo moja limewekwa dijiti kuwa moja)",
```

Comment:
This PR is perfect for now, but we should definitely consider moving all these translations out into a separate YAML or something, at some point.

This way users could update the translations via weblate or transifex (we will add this for UI translations soon anyway)

Reply:
Yeah I also thought about that :+1: